src/backend/data/post.js

```diff
@@ -5,6 +5,7 @@ const textParser = require('../utils/text-parser');
 const Feed = require('./feed');
 const hash = require('./hash');
 const ArticleError = require('./article-error');
+const { indexPost } = require('../lib/elastic');
```

Comment:
We've got our redis `storage` code in `utils/` and elastic in `lib/`.  They seem like they should sit beside each other.